index.js

```diff
@@ -65,24 +89,31 @@
 
 // ***** Question 6 *****
 // *** Uncomment the lines below to test
-// console.log("%cQuestion 6", "color: red")
-
-// const users = [ 
-//   { 
-//     name: "Duane", phones: { cell: "555-123-4567", office: "555-456-7890" }
-//   },
-//   { 
-//     name: "Liza", phones: { cell: "555-234-5678", office: "555-567-1234" }
-//   }
-// ]
-// printNameAndPhones(users)
-// // => "Duane"
+console.log("%cQuestion 6", "color: red")
+
+function print_name_and_phones(users) {
+  users.forEach(function (){})
+  console.log(name)
+  console.log(cell)
+  console.log(office)
+}
```

Comment:
Inside the callback, think about what code you would want to write to console log the key value pairs of just one object. forEach is going to call this anonymous call back function on each item in the `users` array.

```javascript
users.forEach(function (oneUserObject){
  console.log(oneUserObject.name)
  console.log(`Cell: ${oneUserObject.phones.cell}`)
  console.log(`Office: ${oneUserObject.phones.office}`)
})
```